docs/openshift-installation.md

```diff
@@ -0,0 +1,18 @@
+# Openshift Installation
+
+This installation method is the recommended way for Openshift users. **Note**: Openshift version must be 4.2 or higher.
+
+The NGINX Ingress Operator is a [RedHat certified Operator](https://connect.redhat.com/en/partner-with-us/red-hat-openshift-operator-certification).
+
+1. In the Openshift dashboard, click in `Operators` > `Operator Hub` in the left menu and use the search box to type `nginx ingress`:
+![alt text](./images/openshift1.png "Operators")
+1. Click in the NGINX Ingress Operator and click Install:
+![alt text](./images/openshift2.png "NGINX Ingress Operator")
+1. Click in Subscribe (**Note:** We recommend to mark Specific Namespace for the Operator):
+![alt text](./images/openshift3.png "NGINX Ingress Operator Install")
+
+Openshift will install the NGINX Ingress Operator:
+
+![alt text](./images/openshift4.png "NGINX Ingress Operator Subscribe")
+
+You can now deploy the NGINX Ingress Controller instances manually following the [examples](../examples) or using the Openshift Dashboard.
```

Comment:
in that example, should we use "oc" tool or make some kind of comment about the "oc" tool. In the OS docs like this one https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html, "oc" is used.

Reply:
Hmm I think I can add a note?  BTW `kubectl` should work for Openshift as well  (from 4.1 https://docs.openshift.com/container-platform/4.1/cli_reference/usage-oc-kubectl.html#the-kubectl-binary), obviously RH uses `oc` in all their official doc. However the examples were designed for k8s and os, not only os.

Reply:
@Rulox thanks for the clarification. If OS users are used to and ok with `kubectl` commands, than there is no need for adding notes or changes